packages/trace-viewer/src/ui/workbench.tsx

```diff
@@ -225,7 +225,7 @@ export const Workbench: React.FunctionComponent<{
     id: 'network',
     title: 'Network',
     count: networkModel.resources.length,
-    render: () => <NetworkTab boundaries={boundaries} networkModel={networkModel} onEntryHovered={setHighlightedEntry}/>
+    render: () => <NetworkTab boundaries={boundaries} networkModel={networkModel} onEntryHovered={setHighlightedEntry} sdkLanguage={model?.sdkLanguage ?? 'javascript'} />
```

Comment:
i've added a fallback here to make types easier to reason about. this is only unavailable before the model is loaded, so falling back to `javascript` in a loading state is alright I feel.